src/workerd/jsg/buffersource.h

```diff
@@ -208,6 +208,14 @@ class BackingStore {
     return BackingStore(backingStore, byteLength, byteOffset, elementSize, ctor, integerType);
   }
 
+  template <typename T = v8::Uint8Array>
```

Comment:
Nit:

Any particular reason for not using class?

```suggestion
  template <class T = v8::Uint8Array>
```